applications/GeoMechanicsApplication/custom_processes/apply_vector_constraint_table_process.cpp

```diff
@@ -52,16 +57,18 @@ std::vector<char> ApplyVectorConstraintTableProcess::ActiveComponents(const Para
     return result;
 }
 
-Parameters ApplyVectorConstraintTableProcess::CreateParametersForComponent(const Parameters& rSettings, char component)
+Parameters ApplyVectorConstraintTableProcess::CreateParametersForComponent(const ModelPart& rModelPart,
+                                                                           const Parameters& rSettings,
+                                                                           char Component)
 {
     Parameters result;
-    const auto index = ComponentToIndex(component);
-    result.AddValue("model_part_name", rSettings["model_part_name"]);
+    const auto index = ComponentToIndex(Component);
+    result.AddValue("model_part_name", "\"" + rModelPart.Name() + "\"");
```

Comment:
I'm wondering if this would also work without the "\"" around the name:
```suggestion
    result.AddValue("model_part_name", rModelPart.Name());
```
I guess not, but might be worth to try to make the code slightly simpler

Reply:
No, I tried it and it does not work. A strange thing is that it works without "" in other class.

Reply:
Clear, thanks!